### PR TITLE
Added docs and examples for scrollview and section

### PR DESCRIPTION
**Pull request by js-goupil**
### Background

This adds documentation and examples for both ScrollView and Section.

### Solution

(Describe your solution, why this approach was chosen, and what the alternatives/impacts may be)

### 🎩

Copy and paste the code samples and make sure they look good!

Tophat links:

Section: https://shopify-dev.ui-extensions-7t4u.jeansebastien-goupil.us.spin.dev/docs/api/pos-ui-extensions/unstable/components/section

ScrollView: https://shopify-dev.ui-extensions-7t4u.jeansebastien-goupil.us.spin.dev/docs/api/pos-ui-extensions/unstable/components/scrollview

### Checklist

- [ ] I have :tophat:'d these changes
- [ ] I have updated relevant documentation

### CODE REVIEW

File: packages/ui-extensions/docs/surfaces/point-of-sale/reference/types/ExtensionTargetType.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.ExtensionTargetType = void 0;
+var ExtensionTargetType;
+(function (ExtensionTargetType) {
+    ExtensionTargetType["PosHomeModalRender"] = "pos.home.modal.render";
+    ExtensionTargetType["PosHomeTileRender"] = "pos.home.tile.render";
+    ExtensionTargetType["PosPurchasePostActionMenuItemRender"] = "pos.purchase.post.action.menu-item.render";
+    ExtensionTargetType["PosPurchasePostActionRender"] = "pos.purchase.post.action.render";
+})(ExtensionTargetType = exports.ExtensionTargetType || (exports.ExtensionTargetType = {}));
```
Comment (NathanJolly) [inline packages/ui-extensions/docs/surfaces/point-of-sale/reference/types/ExtensionTargetType.js:10]: Can remove this

Comment (js-goupil) [reply packages/ui-extensions/docs/surfaces/point-of-sale/reference/types/ExtensionTargetType.js:10]: wtf

File: packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/Section.doc.ts [hunk 2]
```diff
@@ -1,4 +1,8 @@
 import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+import {generateCodeBlock} from '../helpers/generateCodeBlock';
+
+const generateCodeBlockForComponent = (title: string, fileName: string) =>
```
Comment (vctrchu) [inline packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/Section.doc.ts:4]:
```suggestion
const generateCodeBlockForSection = (title: string, fileName: string) =>
```

This is a nit so up to your discretion for changes

File: packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/ScrollView.doc.ts [hunk 3]
```diff
@@ -1,4 +1,8 @@
 import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+import {generateCodeBlock} from '../helpers/generateCodeBlock';
+
+const generateCodeBlockForComponent = (title: string, fileName: string) =>
```
Comment (vctrchu) [inline packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/ScrollView.doc.ts:4]:
```suggestion
const generateCodeBlockForScrollView= (title: string, fileName: string) =>
```

**Review by vctrchu — APPROVED** [review]
LGTM

**Pull request merged**